### DISCUSSION TITLE
Interested in contribution

### DISCUSSION THREAD

**Discussion by neerajrp1999** [SeaQL/summer-of-code#16] [General]
Hello,
My name is Neeraj Prajapati. I am a master's in computer application Student. I'm learning rust wants to make some Open Source contributions to GSoC.
I'm kind of pretty sure that I'm not going to get selected for GSoC 2023, based on my knowledge of Rust. I have a bit of knowledge of Rust! but I want to make contributions to this project.
I was wondering if I could make a start by contributing to any small task, Let me know, Thank you so much!

**Comment by billy1624** [comment]
Hi @neerajrp1999, thanks for your interest in contributing to SeaQL!! We were a mentor organization in GSoC 2022. However, we were not accepted into GSoC 2023.

With that said, we will offer our own internship program, it will be the 3rd cohort in 2023. An announcement will be made shortly, stay tuned.

**Comment by billy1624** [comment]
Hey @neerajrp1999, please check here for the internship details :)
https://www.linkedin.com/jobs/view/3519852307/
